src/plugins/blocksmith/constants.js

```diff
@@ -22,3 +22,29 @@ export const pbjxChannelNames = {
   POLL_SEARCH: t('pollSearch'),
   VIDEO_SEARCH: t('videoSearch'),
 };
+
+export const blockTypes = {
+  ATOMIC: 'atomic',
+  ORDERED_LIST_ITEM: 'ordered-list-item',
+  UNORDERED_LIST_ITEM: 'unordered-list-item',
+  UNSTYLED: 'unstyled',
+};
+
+export const inlineStyleTypes = {
+  HIGHLIGHT: 'HIGHLIGHT',
+};
+
+export const mutabilityTypes = {
+  MUTABLE: 'MUTABLE',
+  IMMUTABLE: 'IMMUTABLE',
+};
+
+export const entityTypes = {
+  EMOJI: 'EMOJI',
+  LINK: 'LINK',
+};
+
+export const tokens = {
+  BLOCKSMITH_COPIED_CONTENT_TOKEN: 'BLOCKSMITH_COPIED_CONTENT_TOKEN:',
```

Comment:
@arunkarnati just because the clipboard payload ends up looking like this

`BLOCKSMITH_COPIED_CONTENT_TOKEN:[{"_schema":"pbj:tmz:canvas:block:text-block:1-0-0","aside":false,"text":"<p>what</p>"},{"_schema":"pbj:tmz:canvas:block:image-block:1-0-1","etag":"f600eb31d0b28453a620d9f9eff67e89","aside":false,"node_ref":"tmz:image-asset:image_jpg_20200227_564253414766435da00a223a68fa8a9e","is_nsfw":false,"url":"http://www.toofab.com","launch_text":"Socialites Found Embracing in Sunken Car After Fatal Plunge From Ferry","aspect_ratio":"auto","theme":"toofab"},{"_schema":"pbj:tmz:canvas:block:text-block:1-0-0","aside":false,"text":"<p>what</p>"}]`

Reply:
@efjacobson - oh, cool.